web/client/components/map/cesium/Layer.jsx

```diff
@@ -167,7 +167,7 @@ class CesiumLayer extends React.Component {
                 return false;
             }
         }
-        return visibility !== false;
+        return visibility;
```

Comment:
give this change the other change to the config are not needed right? if so we can remove them,  and also remove the migration guidelines changes. but i think it makes also sense to keep them as they will make the app config more coeherent and robust

Anyway changing this will change the default behaviour with 3d layers...

@allyoucanmap since this was a functionality you developed some time ago, do you remember why it was returning that condition "visibility !== false"

Reply:
ok this change is fine, we will treat non defined visibility prop as not rendering